azure_pipeline.yaml

```diff
@@ -6,18 +6,164 @@ trigger:
   branches:
     include:
     - master
-    
+  paths:
+    exclude:
+    - /modules/azure-landing-zone/*
+    - /components/global/*
+
 variables:
   terraformVersion: 0.12.26
   agentPool: 'ubuntu-18.04'
+  build: $(Build.BuildNumber)
+  isMaster: $[eq(variables['Build.SourceBranch'], 'refs/heads/master')]
   action:
 
+parameters:
+  - name: environments
```

Comment:
is environments the right term for this object?

Reply:
changed "environments" to "environment_components"